### ISSUE TITLE
Jaws freezes for a significant amount of time if you hold down arrow keys while swinging leg

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/john-travoltage#447]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Chrome + Jaws
**Problem description**
For https://github.com/phetsims/qa/issues/693. Not in NVDA + Firefox on the same device.
When holding down the arrow keys to continuously swing the leg, when you let go noting happens and Jaws freezes for a significant amount of time. The arrow keys, tab, and shift tab do nothing. Several seconds later Jaws jumps ahead with the missed commands.

**Visuals**

https://user-images.githubusercontent.com/41024075/129254560-94f0e5bd-e836-43d2-8289-c664a40aa76e.MOV

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪John Travoltage‬
URL: https://phet-dev.colorado.edu/html/john-travoltage/1.6.0-rc.2/phet/john-travoltage_all_phet.html
Version: 1.6.0-rc.2 2021-08-10 16:45:44 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
Language: en-US
Window: 1280x657
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe, unfortunately this is something that has been around for a long time. It is a JAWS performance problem/bug and an issue for all sliders on the Web, not specific to PhET. I am pretty sure this is https://github.com/phetsims/qa/blob/master/documentation/accessibility-bugs.md#122-chrome-bugs.

Can you please see if this happens in the published sim too? If so I think that will confirm there is nothing new in the sim causing this and this can be closed.
